qCC/ui_templates/heightGridGenerationDlg.ui

```diff
@@ -83,7 +83,7 @@
         <item>
          <widget class="QComboBox" name="typeOfProjectionComboBox">
           <property name="toolTip">
-           <string>choose the height (z coordinate) projected : maximum = highest point in the cell or average)</string>
+           <string>choose the height (z coordinate) projected : maximum = highest point in the cell (or average)</string>
```

Comment:
I have removed this file yesterday (as it was not used in the project anymore). You'll have to rebase your branch.

Reply:
Got it. I did the rebase of my local branch.